Header Header
string label
int32 label_index

int32 associated_frame
int32 tracking_label

#center (x,y) will be starting x, y which is the bottom left corner of the image
#size_x and size_y will be with and height of image
vision_msgs/BoundingBox2D bounding_box